src/dmd/common/audit_log.d

```diff
@@ -0,0 +1,76 @@
+module dmd.common.audit_log;
+import core.stdc.stdlib;
+import core.stdc.stdio;
+import dmd.root.rootobject;
+import dmd.globals;
+import dmd.ast_node;
+import dmd.hdrgen;
+import dmd.expression;
+import dmd.dsymbol;
+import dmd.statement;
+import dmd.mtype;
+import dmd.init;
+import dmd.dtemplate;
+import dmd.common.outbuffer;
+
+private FILE*[string] file;
+
+void Audit_Log(ASTNode node, string functioncall = __FUNCTION__, string modulecall = __MODULE__)
+{
+    debug(LOGAUDIT)
```

Comment:
log audit or audit log, you've used both.

Reply:
Done